### ISSUE TITLE
invalid codeName attribute for DuplicateKeyException

### ISSUE THREAD

**Issue by asereda-gs** [bwaldvogel/mongo-java-server#59]
When inserting document with duplicate key I get the following error 

```text
Command failed with error 11000 (Location11000): 
'E11000 duplicate key error collection: testDB.entity index: _id_ dup key: { : "e1" }' 
on server localhost:50769. The full response is 
{"$err": "E11000 duplicate key error collection: testDB.entity index: _id_ dup key: { : \"e1\" }", 
"errmsg": "E11000 duplicate key error collection: testDB.entity index: _id_ dup key: { : \"e1\" }",
 "code": 11000, "codeName": "Location11000",
 "ok": 0}
```

MongoDB returns `codeName: DuplicateKey` (compared to `Location11000`)

**Comment by bwaldvogel** [comment]
Error codeNames should be fixed with 58163958702fe414e0e80360cb640645d8d043ce. Can you retest?

**Comment by bwaldvogel** [comment]
The fix is contained in version `1.14.0`.

Please reopen or create a new issue if you it still doesn’t work for you.

**Comment by asereda-gs** [comment]
Thank you, @bwaldvogel, it works.
